be/src/util/bitmap_value.h

```diff
@@ -1451,6 +1451,26 @@ class BitmapValue {
         }
     }
 
+    /**
+     * Return new set with specified range (not include the range_end)
+     */
+    int64_t sub_range(const int64_t& range_start, const int64_t& range_end, BitmapValue* ret_bitmap) {
+        int64_t count = 0; 
+        for (auto it = _bitmap.begin(); it != _bitmap.end(); ++it) {
+            if (*it < range_start) {
+                continue;
+            }
+            if (*it < range_end) {
+                ret_bitmap->add(*it);
+                ++count;
+            } else {
+                break;
+            }
+        }
+        return count;
```

Comment:
If range_start is greater than bitmap max, count will be equal to 0, and finally an empty string will be output, whether to unified into "null"

Reply:
When there is a error, it returns `null`. The case `range_start` greater than `bitmap_max` is a normal case, I think it's ok to return an empty result.